Sources/DistributedActors/Cluster/ClusterShell.swift

```diff
@@ -149,9 +153,13 @@ internal class ClusterShell {
         // notify the failure detector, that we shall assume this node as dead from here on.
         // it's gossip will also propagate the information through the cluster
         traceLog_Remote(system.cluster.uniqueNode, "Finish terminate association [\(remoteNode)]: Notifying SWIM, .confirmDead")
-        if let swim = self._swimRef {
+        if let swim = self._swimShell {
             system.log.warning("Confirm .dead to underlying SWIM, node: \(reflecting: remoteNode)")
-            swim.tell(.local(SWIM.LocalMessage.confirmDead(remoteNode)))
+            Task {
+                await swim.whenLocal { swim in
+                    swim.confirmDead(node: remoteNode)
+                }
+            }
```

Comment:
To make this look a bit less horrible you could

```
nonisolated func confirmDead(node ...) {
  Task { await ... // same as here }
}
```

onto the swim shell, so the call site will look nice: just `swim.confirmDead(...)`

Reply:
It doesn't allow me to:

```
/code/Sources/DistributedActors/Cluster/SWIM/SWIMActorShell.swift:475:34: error: distributed actor-isolated property 'swim' can not be accessed from a non-isolated context
            let directive = self.swim.confirmDead(peer: node.asSWIMNode.swimActor(self.actorSystem))
```

Reply:
I don't understand that one -- `self.swim` is on the shell right?

Reply:
Sorry, I didn't do completely as you prescribed. Got it to work now.

Reply:
Uff okey! Was worried we had some isolation issue -- glad it works :)